test/e2e/csv_e2e_test.go

```diff
@@ -14,6 +14,8 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
 	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	v1Crd "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
nit: this import name is weird. Maybe `v1api` is more fitting, so we have a `v1api.CustomResourceDefinition` type in the code?

Reply:
Thanks, fixed it.